server/src/d.ts

```diff
@@ -0,0 +1,223 @@
+import { any } from "underscore";
+
+export type Headers = {
+  [key: string]: any;
+  host?: string;
+  referrer?: string;
+  origin?: string;
+  "x-request-id"?: string;
+  "user-agent"?: string;
+  authorization?: string;
+  "x-polis"?: string;
+  "accept-language"?: string;
+  "Accept-Language"?: string;
+};
+
+export type DetectLanguageResult = {
+  language: string;
+  confidence: any;
+};
+
+export type Body = {
+  [key: string]: any;
+  agid?: any;
+  xid?: any;
+  uid?: any;
+};
+
+export type Query = { [x: string]: any };
+
+export type AuthBody = {
+  x_profile_image_url?: any;
+  x_name?: any;
+  x_email?: any;
+  agid?: any;
+};
+
+export type AuthQuery = {
+  x_profile_image_url: any;
+  x_name: any;
+  x_email: any;
+  agid: any;
+};
+
+export type AuthRequest = {
+  body: AuthBody;
+  query?: AuthQuery;
+};
+
+export type ParticipantInfo = {
+  parent_url?: string;
+  referrer?: string;
+};
+
+export type PidReadyResult = {
+  modOptions?: any;
+  nextComment?: any;
+  currentPid?: any;
+  shouldMod?: any;
+};
+
+export type CommentOptions = {
+  currentPid?: any;
+};
+
+export type CommentType = {
+  zid: any;
+  not_voted_by_pid: any;
+  include_social?: any;
+  withoutTids: any;
+  tid?: any;
+  translations?: any;
+  txt?: any;
+  include_voting_patterns: any;
+  modIn: boolean;
+  pid: any;
+  tids: any;
+  random: any;
+  limit: any;
+  moderation: any;
+  strict_moderation: any;
+  mod: any;
+  mod_gt: any;
+};
+
+export type ParticipantFields = {
+  show_translation_activated?: any;
+};
+
+export type ParticipantCommentModerationResult = {
+  nextComment?: any;
+  currentPid?: any;
+};
+
+// TODO rename this to User after converting
+// TODO User import in server.ts to camelCase
+// TODO in standalone change
+export type UserType = {
+  email?: any;
+  hname?: any;
+  stripeCustomerId?: any;
+  uid?: any;
+  pid?: any;
+  id?: any;
+  screen_name?: any;
+  name?: any;
+  followers_count?: number;
+  friends_count?: number;
+  verified?: any;
+  profile_image_url_https?: string;
+  location?: any;
+  context_id?: any;
+  user_id?: any;
+  user_image?: any;
+  tool_consumer_instance_guid?: any;
+  lis_person_contact_email_primary?: any;
+  lis_person_name_full?: any;
+};
+
+// TODO rename this to Conversation after converting
+// TODO User import in server.ts to camelCase
+// TODO in standalone change
+export type ConversationType = {
```

Comment:
This is called `ConversationType` since we already have a module named `Conversation`.  If, in the future, we rename that module to the camelCase lowercase `conversation`, then we can call this type just `Conversation`. 

This pattern of  a matching`camelCase` variable name and `PascalCase` type name is the convention that I normally follow in TypeScript projects.